server/src/routes/api/users.js

```diff
@@ -160,27 +160,53 @@ router.post('/:id/coursehistory', async (req, res) => {
  */ 
 
 router.get('/:id/coursehistory', async (req, res) => {
-  let userId = req.params.id;
+  let userId = 1;
 
   if (await user.getCourses(userId) <= 0){
     console.log('no course history found for user');
     res.status(200).send('no course history found for user');
   } else {
     const courses = await user.getCourses(userId); 
-    console.log(courses);
-    res.status(200).send("fetching all user courses: " + courses);
+    console.log(courses[0]);
+    res.status(200).send({message: "fetching all user course history", course: courses});
```

Comment:
property should match quantity being send, so if we are sending an array of courses the property should be `courses`